### ISSUE TITLE
Connection Problem

### ISSUE THREAD

**Issue by ChristianFJung** [zhuowei/Xenologer#3]
@zhuowei  I can not access google or send a text . It says connection problem but It syncs my contacts. 

Also the hangout modded file won't download. Can you post a diffrent link.

**Comment by achalv** [comment]
@jungletech Could you please tell me how to sync my contacts/make my Glass app able to text/hangout with people? Mine can access the internet just find. Using a Galaxy Nexus running 4.2.1.

**Comment by ChristianFJung** [comment]
It is pretty easy, you download glass setup moded. Go google.com/myglass and then you hit enter on the glass setup.

Zhowei has better instructions on  setup. But after that. What u do is look at the page where it shows the apps and at the top right it says add contact and you type in there name and it finds it from google contacts.

How did you get a connection?

Was I clear on adding if not just say something and I will explain better.

I prefer the website over the app. But it works either way.

----- Reply message -----

**Comment by leejaew** [comment]
@zhuowei I'm also getting "No contacts found" message when trying to use the voice commands "send a message to" and "make a call to". Can't quite understand the instructions shared by @jungletech could you please elaborate more on how you got the contacts to be synced? Are all the steps taken through the website google.com/myglass?

**Comment by achalv** [comment]
@jungletech Didn't really get what you mean. Step by step instructions, please? :(

**Comment by zhuowei** [comment]
I haven't gotten MyGlass contacts to work for myself yet; try tonystark23's tutorial at https://www.youtube.com/watch?v=fNsu9xvpdNw

**Comment by ChristianFJung** [comment]
@leejaew @achalv @zhuowei 
1. Go to google.com/my glass
2. Go throughout the first steps.
3. Now you get to the qr code, and I figured out the problem , I had it several times
   4 hit enter
   5 exit out of google glass setup
   6, swipe down notifications you should see something. That says permission to use google account. Click it and say yes.

I spent probably an hour focusing on that.

How do I get an Internet connection?

**Comment by zhuowei** [comment]
Try downloading the latest GlassHome, which removes a connectivity check.

**Comment by ChristianFJung** [comment]
@zhuowei  I still get the error and just redownloaded glass home

**Comment by zhuowei** [comment]
Does Google Now or Google Voice Search work on your device? I really don't know what's going on.

**Comment by ChristianFJung** [comment]
Yes it does work maybe this is googles way of preventing us from using glass
